WebContent/WEB-INF/jsp/usersProfiles.jsp

```diff
@@ -133,6 +133,10 @@
     }
     
     function showUserProfile(userProfileId) {
+        if (userProfileId == -1)
```

Comment:
1. Click on 'Add user profile';
2. Empty 'Profile name';
3. Click on 'save'; -> Message appears 'Profile name must be unique and not empty'
4. "Delete" button appears;
5. Click on "Delete" button -> "Are you sure you want to delete this profil?" ok -> "Profile successfully deleted." ok -> "Failed to execute 'removeChild' on 'Node': parameter 1 is not of type 'Node'." error